### PR TITLE
Enable passing answer to download/overwrite question in _upgradePublisher.sh

### PR DESCRIPTION
**Pull request by peturv**
Adds the flags -f|--force to _updatePublisher.sh to allow for a forced download. Usable in non-interactive settings.

See issue #327

### CODE REVIEW

**Comment by cmoesel** [comment]
Hi @peturv -- thanks for the PR!

We actually base our scripts on the ones in HL7's official [sample-ig](https://github.com/FHIR/sample-ig) -- so I would suggest you also make a PR for this there.  We're trying to stay in basic alignment.

I don't suppose you know how to do this in Windows `.bat` as well, do you?  We're also trying to keep parity between the Windows and Mac scripts.  If you don't, someone from my team might be able to do it.

**Comment by peturv** [comment]
> Hi @peturv -- thanks for the PR!
> 
> We actually base our scripts on the ones in HL7's official [sample-ig](https://github.com/FHIR/sample-ig) -- so I would suggest you also make a PR for this there. We're trying to stay in basic alignment.
> 
> I don't suppose you know how to do this in Windows `.bat` as well, do you? We're also trying to keep parity between the Windows and Mac scripts. If you don't, someone from my team might be able to do it.

Thanks for the response!
Unfortunately I don't have a windows-based computer at my disposal at the moment since I am in isolation and not allowed in the office :D

I'll post the PR to the sample project instead

File: src/ig/files/_updatePublisher.sh [hunk 1]
```diff
@@ -31,15 +45,17 @@ else
...
+  else
+    echo Will place publisher jar here: "$jarlocation"
+    message="Ok? (Y/N) "
+  fi
+  read -r -p "$message" response
 fi
 
-read -r -p "$message" response
-if [[ "$response" =~ ^([yY])$ ]]; then
+if [[ "$FORCE" ]] || [[ "$response" =~ ^([yY])$ ]]; then
```
Comment (jafeltra) [inline src/ig/files/_updatePublisher.sh:58]: Hi @peturv! I was reviewing this pull request and noticed one thing that should be changed. I think the check for the force flag here should be checking `"$FORCE" == true`. As it is now, if a user is not using the `--force` flag and chooses `N` at the prompt, the download will still occur. Adding this line should fix this issue.

Comment (peturv) [reply src/ig/files/_updatePublisher.sh]: Thanks! Feel a bit silly having missed that. Have posted an update.

Comment (jafeltra) [reply src/ig/files/_updatePublisher.sh]: No worries! Thanks for pushing a fix!

**Pull request merged**